tests/rules/test_empty_values.py

```diff
@@ -258,3 +258,60 @@ def test_in_flow_mappings_comments(self):
                    problem1=(4, 7),
                    problem2=(7, 9),
                    problem3=(10, 5))
+
+    def test_in_list_items_disabled(self):
+        conf = ('empty-values: {forbid-in-block-mappings: false,\n'
+                '               forbid-in-flow-mappings: false,\n'
+                '               forbid-in-list-items: false}\n'
+                'braces: disable\n'
+                'commas: disable\n')
```

Comment:
Are these needed?

I guess you copied-pasted them from an example with *flow mappings*, which use `{` and `,` characters. Block sequences don't :)

Reply:
sorry, you're right, adjusted.